### DISCUSSION TITLE
请问上传的文件有大小限制吗，最多能上传多大的文件当做索引呢？

### DISCUSSION THREAD

**Discussion by kky529999** [GaiZhenbiao/ChuanhuChatGPT#438] [General]
我有多个文件，大约加起来有500多M，这么大的文件能当做索引吗？

**Comment by GaiZhenbiao** [comment]
理论上可以，实际上……这会消耗很多钱，并且CPU得处理好一会儿。但是，hey，你可以这么干。

**Reply by raines1220** [reply]
想问一下，这个“索引”的功能是默认永远使用“gpt-3.5-turbo-0301”这个模型吗？
我试了下换成gpt-4好像没有什么区别？

**Reply by Git179822868** [reply]
> 理论上可以，实际上……这会消耗很多钱，并且CPU得处理好一会儿。但是，hey，你可以这么干。
请问索引功能只能用来回答文件的问题吗
